src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -3277,14 +3276,12 @@ def cyclic_right_subideals(self, p, alpha=None):
             ans.append(J)
         return ans
 
-    def is_integral(self):
+    def is_integral(self) -> bool:
         r"""
-        Check if a quaternion fractional ideal is integral. An ideal in a quaternion algebra is
-        said integral if it is contained in its left order. If the left order is already defined it just
-        check the definition, otherwise it uses one of the alternative definition of Lemma 16.2.8 of
-        [Voi2021]_.
-
-        OUTPUT: a boolean.
+        Check whether a quaternion fractional ideal is integral. An ideal in a quaternion algebra
+        is integral if and only if it is contained in its left order. If the left order is already
+        defined this method just checks this definition, otherwise it uses one of the alternative
+        definitions from Lemma 16.2.8 of [Voi2021]_.
```

Comment:
```suggestion
        Check whether a quaternion fractional ideal is integral.

        An ideal in a quaternion algebra is integral if and only if it is
        contained in its left order. If the left order is already defined
        this method just checks this definition, otherwise it uses one
        of the alternative definitions from Lemma 16.2.8 of [Voi2021]_.
```
We like 1-line descriptions to be one line (well, really sentence).